test/check-cli

```diff
@@ -49,6 +49,9 @@ class TestTar(composertest.ComposerTestCase):
     def test_tar_kickstart(self):
         self.runCliTest("/tests/cli/test_compose_tar_kickstart.sh")
 
+class TestPylint(composertest.ComposerTestCase):
+    def test_ci(self):
+        self.runCliTest("/tests/cli/test_ci.sh")
```

Comment:
This must be `runImageTest()` after cherry-picking the changes in #871, at least the 1st commit which makes it possible to run anything inside the VM without starting lorax-composer.

Otherwise we're starting lorax-composer before running `make ci` and it likely causes the composer process started by the Python test suite to fail.